admin/resources/js/components/pool/PoolPage.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { defineMessages, useIntl } from "react-intl";
+import { Link } from "../../helpers/router";
+
+const messages = defineMessages({
+  poolCreateHeading: {
+    id: "operationalRequirementPage.poolCandidateCreateHeading",
```

Comment:
Change `operationalRequirementPage.poolCandidateCreateHeading` to `poolPage.poolCreateHeading`